### DISCUSSION TITLE
Now I understand how Copilot important for me. so, when will the copilot come back

### DISCUSSION THREAD

**Discussion by ryan-minato** [community/community#40033] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

I have a valid copilot's license, but plugin tell me **"GitHub Copilot: Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended. // Signup for GitHub Copilot // Dismiss"** .

I tried in PyCharm, Webstrom(jetbrains) and vscode(ms). 

- [x] Log-out
- [x] Reinstall
- [x] Log-in.

But it did not work.

I do believe it is a register server issue.

**Comment by davecheney** [comment, marked as answer]
Hello. I'm sorry you had difficulty accessing Copilot. This was caused by a bad deploy which affected a small percentage of Copilot users. The change has been rolled back and you should be able to use Copilot again.
